pkg/kubelet/kubelet.go

```diff
@@ -460,6 +468,42 @@ func NewMainKubelet(
 			return nil, err
 		}
 		klet.containerRuntime = rktRuntime
+	case "remote":
```

Comment:
Here, `remote` is only used for switch..case, the real container runtime is got by `Version()` API. Check https://github.com/kubernetes/kubernetes/pull/28396/files#diff-8da51ab012ecbdf2b013ecbcf1c37033R114 and https://github.com/kubernetes/kubernetes/pull/28396/files#diff-8da51ab012ecbdf2b013ecbcf1c37033R141